src/components/Hero/HomeHero/index.tsx

```diff
@@ -2,25 +2,29 @@ import * as React from "react"
 import { useTranslation } from "react-i18next"
 import { Box, Heading, Stack, Text, VStack } from "@chakra-ui/react"
 
+import { Image } from "@/components/Image"
+
 import { ButtonLink } from "../../Buttons"
-import GatsbyImage from "../../GatsbyImage"
 import Morpher from "../../Morpher"
 import Translation from "../../Translation"
 import { CommonHeroProps } from "../utils"
 
 export interface HomeHeroProps extends Pick<CommonHeroProps, "heroImgSrc"> {}
 
 const HomeHero = ({ heroImgSrc }: HomeHeroProps) => {
-  const { t } = useTranslation()
+  const { t } = useTranslation("common")
```

Comment:
should be `"page-index"`here instead of `"common"`? and  `"page-index"` could be removed maybe